Calendar.ASP.NET.MVC5/App_Start/MyClaimTypes.cs

```diff
@@ -0,0 +1,28 @@
+﻿/*
+Copyright 2015 Google Inc
+
+Licensed under the Apache License, Version 2.0(the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace Calendar.ASP.NET.MVC5
+{
+    internal static class MyClaimTypes
+    {
+        public const string GoogleUserId = "GoogleUserId";

```

Comment:
This is a key we use to associate the user's Google user ID with their ASP.NET user account. The access/refresh tokens are kept in the API client's data store, indexed by Google user ID.